test/src/test/java/hudson/model/ItemsTest.java

```diff
@@ -185,7 +190,7 @@ private void cannotOverwrite(String target) throws Exception {
                 JenkinsRule.WebClient wc = wc(r);
                 wc.getOptions().setRedirectEnabled(false);
                 wc.getOptions().setThrowExceptionOnFailingStatusCode(false); // redirect perversely counts as a failure
-                WebResponse webResponse = wc.getPage(new WebRequest(createCrumbedUrl(r, wc, "createItem?name=" + target + "&mode=hudson.model.FreeStyleProject"), HttpMethod.POST)).getWebResponse();
+                WebResponse webResponse = wc.getPage(new WebRequest(new URL(wc.getContextPath() + "createItem?name=" + target + "&mode=hudson.model.FreeStyleProject"), HttpMethod.POST)).getWebResponse();
```

Comment:
BTW we could probably introduce `WebClient.postTo(String)` (or something) as a convenience.